ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -122,6 +122,19 @@ tests:
     test:
     - ref: openshift-extended-test
     - ref: rosa-terraform-e2e-tests-upload-polarion
+- as: rosa-sts-ad-critical-high-f3
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
+  steps:
+    cluster_profile: aws-qe
```

Comment:
cluster_profile should be "aws-terraform-qe"

Reply:
"aws-terraform-qe" cannot work stable. I will use aws-qe before I figuring out why  "aws-terraform-qe" sometimes work but sometimes not.

Reply:
@xueli181114  it is regular AWS user with Admin permissions (as we added it) if it misses some permission - let's fix it

Reply:
aws-terraform-qe, has all the right credentials to run (see current periodics executions: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-terraform-redhat-terraform-provider-rhcs-main-managed-oidc-config-e2e-periodic/1719194833353969664/artifacts/)

You need to be able to use specific Az (like ^^)

Reply:
It's not related to the AZs. It sometimes fail, sometimes succeed. The possible reason is resource limitation. Need to debug into it when I have time. Or @amalykhi you can use the AWS secret locally to check what happened when it fail. But I will keep aws_qe to make sure the CI run stable before we find out the root cause

Reply:
@xueli181114 Yes it is. resources are limited in us-east-1 region with aws-terraform-qe.  therefore we are using this cluster_profile with region=ap-northeast-1 - and we have stable runs for almost 5 months now.